pkg/document/testdata/doc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "@context": "https://w3id.org/did/v1",
+  "@context": ["https://w3id.org/did/v1"],
```

Comment:
we may need additional test if `"@context": "https://w3id.org/did/v1"` is also legal.

Reply:
spec say is array

Reply:
@troyronda @sandrask @rolsonquadras  https://github.com/hyperledger/aries-framework-go/blob/master/pkg/doc/did/doc.go#L48

Reply:
I will create issue

Reply:
@troyronda https://github.com/trustbloc/sidetree-core-go/issues/52